samples/azure/polymorphic-events-asb/Version_6/Shared/ConfigErrorQueue.cs

```diff
@@ -0,0 +1,13 @@
+﻿using NServiceBus.Config;
+using NServiceBus.Config.ConfigurationSource;
+
+class ConfigErrorQueue : IProvideConfiguration<MessageForwardingInCaseOfFaultConfig>
+{
+    public MessageForwardingInCaseOfFaultConfig GetConfiguration()
```

Comment:
This _is_ code config ;) Looks like we've got issue with naming things.
Perhaps you mean to use 
```c#
busConfiguration.SendFailedMessagesTo("error");
busConfiguration.AuditProcessedMessagesTo("audit");
```